client/src/App/App.tsx

```diff
@@ -93,6 +96,59 @@ const App: React.FC = () => {
                 <ConnectedUserCredentials {...props} keycloakBaseURL={KEYCLOAK_API_BASE_URL} />
               )}
             />
+            <Route
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={false}
+              exact
+              path={URL_LOCATIONS_UNIT}
+              render={(props: any) => {
+                const tableData: data[] = [];
+                for (let i = 1; i < 5; i++) {
+                  tableData.push({
+                    key: i.toString(),
+                    name: `Edrward ${i}`,
+                    level: i,
+                    lastupdated: new Date(
+                      `Thu Oct ${i} 2020 14:15:56 GMT+0500 (Pakistan Standard Time)`
+                    ),
+                    status: 'Alive',
+                    type: 'Feautire',
+                    created: new Date(
+                      `Thu Oct ${i} 2020 14:15:56 GMT+0500 (Pakistan Standard Time)`
+                    ),
+                    externalid: `asdkjh123${i}`,
+                    openmrsid: `asdasdasdkjh123${i}`,
+                    username: `edward ${i}`,
+                    version: `${i}`,
+                    syncstatus: 'Synced',
+                  });
+                }
+
+                const tree: TREE[] = [
+                  {
+                    title: 'Sierra Leone',
+                    key: 'Sierra Leone',
+                    children: [
+                      { title: 'Bo', key: 'Bo', children: [{ title: '1', key: '1' }] },
+                      { title: 'Bombali', key: 'Bombali', children: [{ title: '2', key: '2' }] },
+                      {
+                        title: 'Bonthe',
+                        key: 'Bonthe',
+                        children: [
+                          {
+                            title: 'Kissi Ten',
+                            key: 'Kissi Ten',
+                            children: [{ title: 'Bayama CHP', key: 'Bayama CHP' }],
```

Comment:
What is this stuff for?

Reply:
Thats the dummy data to be feed into the component later on (the on i am starting to work on) this will be fed by connected reducers

Reply:
I'm not sure that we should merge this with dummy data.

Reply:
@moshthepitt we should get #52 mergerd first then and i connect this with reducer and impliment real data ?

Reply:
I just merged #52 @ashfahan